libs/aws/langchain_aws/chat_models/bedrock.py

```diff
@@ -439,7 +439,7 @@ def _stream(
         if "claude-3" in self._get_model():
             if _tools_in_params({**kwargs}):
                 result = self._generate(
-                    messages, stop=stop, run_manager=run_manager, **kwargs
+                    messages, stop=stop, run_manager=run_manager, stream=False, **kwargs
```

Comment:
Isn't this hardcoding and disabling streaming for all tool calls? I might be missing some context here, is there a documented issue regarding this in Bedrock/Anthropic or documentation that refers to this?